src/misc/lv_text.h

```diff
@@ -75,39 +91,33 @@ typedef enum {
  * @param size_res pointer to a 'point_t' variable to store the result
  * @param text pointer to a text
  * @param font pointer to font of the text
- * @param letter_space letter space of the text
- * @param line_space line space of the text
- * @param max_width max width of the text (break the lines to fit this size). Set COORD_MAX to avoid
- * @param flag settings for the text from ::lv_text_flag_t
-
- * line breaks
+ * @param attributes the text attributes, flags for line break behaviour, spacing etc
  */
-void lv_text_get_size(lv_point_t * size_res, const char * text, const lv_font_t * font, int32_t letter_space,
-                      int32_t line_space, int32_t max_width, lv_text_flag_t flag);
+void lv_text_get_size(lv_point_t * size_res, const char * text, const lv_font_t * font,
```

Comment:
I've just checked that it seems possible to hide these functions in `lv_text_private.h` (keep the  enums here)

It would make it clear that these re internal and can be changes at any time.